### PR TITLE
Add Simulcast support to RtpPacketizer

### LINKED ISSUE #891: Simulcast Sender
Hey @paullouisageneau 

I would like to add the ability to send Simulcast from libdatachannel. Would you accept this feature? Happy to type up justifications if you aren't sure if it is worth it yet. Do you have any opinions on API? I see these options.

### Add a new constructor to `Media`
Users could construct multiple `Media`s with different RIDs. Then would call `Send` as usual.

```
rtc::Description::Video highQuality("video", rtc::Description::Direction::SendOnly, "h");
rtc::Description::Video mediumQuality("video", rtc::Description::Direction::SendOnly, "m");
rtc::Description::Video lowQuality("video", rtc::Description::Direction::SendOnly, "l");

auto highQualityTrack = pc->addTrack(highQuality);
highQualityTrack->send(buffer, len);
```

### Express multiple RIDs on one Track or Media
```
rtc::Description::Video video("video", rtc::Description::Direction::SendOnly);

auto highRid = video->addRid("h");
auto mediumRid = video->addRid("m");
auto lowRid = video->addRid("l");

auto videoTrack = pc->addTrack(video);
videoTrack->send(buffer, len, highRid);
```

### PR DESCRIPTION
**Pull request by Sean-Der**
Resolves #891

### CODE REVIEW

**Comment by Sean-Der** [comment]
@pkviet ported my changes to C++ ! After this is merged+tagged I would like to pull this in and get Simulcast support into OBS

@paullouisageneau I tried to match the existing API closely. I don't feel strongly about what the API looks like, but feels 'best' to match what we have today :) 

thanks! and sorry for the Churn

File: include/rtc/rtppacketizationconfig.hpp
Comment (Sean-Der) [inline include/rtc/rtppacketizationconfig.hpp:21]: Allows me to change the SSRC as I move through the layers

File: include/rtc/rtppacketizationconfig.hpp
Comment (paullouisageneau) [inline include/rtc/rtppacketizationconfig.hpp:59]: I think this should be a `string`, or better an `optional<string>`.

**Comment by paullouisageneau** [comment]
> @pkviet ported my changes to C++ ! After this is merged+tagged I would like to pull this in and get Simulcast support into OBS

It sounds great, thanks for the PR!

> @paullouisageneau I tried to match the existing API closely. I don't feel strongly about what the API looks like, but feels 'best' to match what we have today :)

It looks good overall, I just have comments on a couple details.

**Comment by Sean-Der** [comment]
@paullouisageneau fixed! Could I get another review please, thank you!

**Review by paullouisageneau — APPROVED** [review]
It looks good, thanks!

**Pull request merged**
